src/test/java/jdbchomework/entity/AbstractEntityTest.java

```diff
@@ -0,0 +1,58 @@
+package jdbchomework.entity;
```

Comment:
Please move Unit tests to another Pull request. Or move Dao optimization (removal of transactions) to another PR